src/couch_jobs/src/couch_jobs_fdb.erl

```diff
@@ -33,6 +33,7 @@
 
     get_activity_vs/2,
     get_activity_vs_and_watch/2,
+    get_active_jobs/1,
```

Comment:
This is not needed anymore? We moves the function to `couch_jobs`

Reply:
ya was rebasing to get the couch_jobs addition as a separate commit and forgot to undo this